ComputeClient/Compute.Client/ApiUris.cs

```diff
@@ -1,5 +1,5 @@
 ﻿using System.Data;
```

Comment:
formated file using Visual Studio 2017 - real changes in this file start on line 3039